src/frontend/utils/dato.ts

```diff
@@ -30,5 +38,38 @@ export const formatterDateTilIsoString = (dato?: Date): string =>
 export const formatterDateTilIsoStringEllerUndefined = (dato?: Date): string | undefined =>
     dato && isValid(dato) ? format(dato, Datoformat.ISO_DAG) : undefined;
 
+interface FormatterIsoStringProps {
+    isoDatoString: IsoDatoString | undefined;
+    tilFormat: Datoformat;
+    defaultString?: string;
+}
+
+export const formatterIsoDatoString = ({
+    isoDatoString,
+    tilFormat,
+    defaultString = '',
+}: FormatterIsoStringProps): string => {
+    const dato = isoDatoString ? new Date(isoDatoString) : undefined;
+    return formatterDate({ dato: dato, datoformat: tilFormat, defaultString: defaultString });
+};
+
+export const parseIsoString = (isoDatoString: IsoDatoString): Date => {
```

Comment:
Er dette navnet ok eller burde den heller hete noe ala `isoStringTilDate`?

Reply:
Likte bedre isoStringTilDate. Da sier vi hva vi har og hva vi endrer til uten at navnet blir for langt